src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -546,42 +546,54 @@ public function attendance_totals() {
 	 *
 	 * @since 4.7
 	 *
-	 * @param array  $data
-	 * @param string $order_id
-	 * @param int    $event_id
+	 * @param array $attendee_data Information that we are trying to save.
+	 * @param int   $attendee_id   The attendee ID.
+	 * @param int   $post_id       The event/post ID.
 	 */
-	public function update_attendee_data( $data, $order_id, $event_id ) {
+	public function update_attendee_data( $attendee_data, $attendee_id, $post_id ) {
+		// Bail if the user is not logged in.
+		if ( ! is_user_logged_in() ) {
+			return;
+		}
```

Comment:
Capability check?

Reply:
Further below in the logic is makes sure the attendee matches one 'owned' by the current user ID.